### PR TITLE
added an example react todos app using hooks, updated the review changes

### PR DESCRIPTION
**Pull request by nik72619c**
also added changes in reference to #11 
changes implemented :-

Use hooks to store todo Input.
TodoApp.css -> style.css
TodoView.jsx -> component.jsx

@TheSTL @ashokdey please have a look.

### CODE REVIEW

File: src/app/TodoApp/container.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import TodoView from './component';
+import './style.css';
+function TodoApp() {
+  // Declare a new state variable, which we'll call "count"
+  const [todos, setTodo] = useState(['your sample todo']);
+  const [todoInput, setTodoInput] = useState("");
+  return (
+    <div>
+        <input type="text" name="addtodo" id="todoInput" placeholder="Enter your todos" onChange={(event)=>setTodoInput(event.target.value)}/>
```
Comment (TheSTL) [inline src/app/TodoApp/container.jsx]: Can you use value attribute ?
Example: 
value = { todoInput }

File: src/app/TodoApp/container.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <button className="add-button" onClick={
+            ()=>{
+                console.log(document.getElementById("todoInput").value);
+                setTodo([...todos, todoInput && todoInput]);
+                document.getElementById("todoInput").value = "";
```
Comment (TheSTL) [inline src/app/TodoApp/container.jsx]: I think it would be nice of you use todoInput instead of document.getElementById("todoInput").value .

**Comment by nik72619c** [comment]
okay, wiil implememt these changes too

**Comment by nik72619c** [comment]
working on it!

**Comment by nik72619c** [comment]
implemented the required changes @TheSTL @ashokdey please have a look !

**Pull request merged**